src/tests/test_analysis_vscode.py

```diff
@@ -49,3 +51,4 @@ def test_analysis_quarkus(setup_vscode):
     assert vscode.is_analysis_complete()
     vscode.open_report_page()
     assert vscode.verify_story_points(target="quarkus")
+    vscode.delete_config_files()
```

Comment:
I see that this is part of clean up .
So it can be included in the fixture setup_vscode so that whenever you call the fixture , after the test , clean up code will be called automatically.

Example : https://github.com/windup/windup_integration_test/blob/main/mta/fixtures/project_fixture.py#L25

Reply:
Yes, the right way way of handling the deletion would be through a fixture. 

But, the config files are currently  not set up  through a fixture . So, the code needs some refactoring .